molecule/__init__.py

```diff
@@ -18,19 +18,20 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-import signal
-import sys
+from signal import SIGINT
+from signal import signal
+from sys import exit
 
 from pbr import version
 
 
 def signal_handler(signal, frame):
     msg = '{red}Exiting molecule on Ctrl+c.{reset}'
     print(msg.format(red='\033[31m', reset='\033[0m'))
-    sys.exit()
+    exit()
```

Comment:
Well won't we still get stack traces depending on when ^C is pressed? If we don't press it when imports are occurring in init, whats to say we wont get them when imports are loaded elsewhere?